### DISCUSSION TITLE
How to use closedlist plug in

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#93] [Q&A]
How to use closedlist plug in 
https://closedlistdemo.tiddlyhost.com/

**Comment by buggyj** [comment]
first you dock to the tiddlywiki then you dock to the closedlist, and clips appear inside the closedlist

**Reply by arunnbabu81** [reply]
> first you dock to the tiddlywiki then you dock to the closedlist, and clips appear inside the closedlist

What do you mean by dock to closedlist.

**Reply by buggyj** [reply]
If  you press the button 'dock' on the closedlist tiddler (after docking to the tiddlywiki) it will re-dock to that closed lists and clips will appear inside the closed list

**Reply by arunnbabu81** [reply]
> If you press the button 'dock' on the closedlist tiddler (after docking to the tiddlywiki) it will re-dock to that closed lists and clips will appear inside the closed list

![image](https://user-images.githubusercontent.com/67494083/159002720-132e8e96-3efa-42e0-9770-d198090efb57.png)


This Dock option is not appearing for me in my  wiki.
